pkg/eventmanager/eventmanager.go

```diff
@@ -29,12 +29,14 @@ const (
 	UPGRADE_EXTDEPCHECK_DELAY_DESC = "Cluster upgrade to version %s is experiencing a delay as an external dependency of the upgrade is currently unavailable. The upgrade will continue to retry. This is an informational notification and no action is required by you."
 	// UPGRADE_SCALE_DELAY_DESC describes the upgrade scaling delayed
 	UPGRADE_SCALE_DELAY_DESC = "Cluster upgrade to version %s is experiencing a delay attempting to scale up an additional worker node. The upgrade will continue to retry. This is an informational notification and no action is required by you."
+	// UPGRADE_SCALE_DELAY_SKIP_DESC describes the upgrade scaling skipped after delay
+	UPGRADE_SCALE_DELAY_SKIP_DESC = "Cluster upgrade to version %s is experiencing a delay attempting to scale up an additional worker node. The upgrade will continue without scaling up extra nodes. This is an informational notification and no action is required by you."
```

Comment:
@wgordon17 Could you help check if the message here is proper?

Reply:
```suggestion
	UPGRADE_SCALE_DELAY_SKIP_DESC = "Cluster upgrade to version %s has experienced an unforseen issue during capacity reservation efforts. The upgrade will continue without extra compute. This is an informational notification and no action is required by you."
```